test/ConfCrypt/Commands/Tests.hs

```diff
@@ -59,7 +60,7 @@ modifyFileProperties = testGroup "modify file properties" [
 
     ]
 
-bufferWriteProperties :: TestTree
+{-bufferWriteProperties :: TestTree
```

Comment:
In theory this should still be testable using the exact same logic, minus the `execWriter` call. Please restore the test.

Reply:
Test restored in last commit.